students/alinafe/codingBat/caught_speeding.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+"""
+You are driving a little too fast, and a police officer stops you. Write code to compute the result, encoded as an int value: 0=no ticket, 1=small ticket, 2=big ticket. If speed is 60 or less, the result is 0. If speed is between 61 and 80 inclusive, the result is 1. If speed is 81 or more, the result is 2. Unless it is your birthday -- on that day, your speed can be 5 higher in all cases.
+"""
+
+def caught_speeding(speed, is_birthday):
+  if is_birthday:
+    speed -= 5
+     
+  if speed <= 60:
+      return 0
+  if 60 < speed <= 80:
```

Comment:
You've already checked that speed > 60 -- so this could be simpler.

not a big deal though -- sometimes repetition is better for clarity.